### PR TITLE
Add wolfSSL_CTX_load_verify_chain_buffer_format

### PR DESCRIPTION
**Pull request by embhorn**
New API to load a DER cert chain for the CA's from a buffer. Test case uses new CA chain buffer.

### CODE REVIEW

File: wolfssl/certs_test.h [hunk 1]
```diff
@@ -1471,6 +1471,228 @@ static const unsigned char ca_cert_der_2048[] =
 };
 static const int sizeof_ca_cert_der_2048 = sizeof(ca_cert_der_2048);
 
+static const unsigned char ca_cert_chain_der[] =
```
Comment (dgarske) [inline wolfssl/certs_test.h]: This file is generated from the `./gencertbufs.pl`, so this new `ca_cert_chain_der` will be overwritten. For this to be easily maintained we need to add something in `./gencertbufs.pl` to combine two ca certs and output to `wolfssl/certs_test.h`.

File: doc/dox_comments/header_files/ssl.h [hunk 2]
```diff
@@ -6904,6 +6904,60 @@ WOLFSSL_API int wolfSSL_CTX_trust_peer_buffer(WOLFSSL_CTX*,
...
+    _Example_
+    \code
+    int ret = 0;
+    int sz = 0;
+    WOLFSSL_CTX* ctx;
+    byte certBuff[...];
+    ...
+
+    ret = wolfSSL_CTX_load_verify_chain_buffer_format(ctx, certBuff,
+                                                      sz, SSL_FILETYPE_PEM);
```
Comment (dgarske) [inline doc/dox_comments/header_files/ssl.h]: We might consider using `WOLFSSL_FILETYPE_ASN1` for the example here to demonstrate load a chain of DER buffers.

Comment (embhorn) [reply doc/dox_comments/header_files/ssl.h]: Check

**Comment by embhorn** [comment]
retest this please

**Comment by embhorn** [comment]
I will reopen after investigating reported address sanitizer failure with DER chain.

**Pull request closed without merging**